webapp/composer.json

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "domjudge/domjudge",
+	"version": "8.4.0DEV",
```

Comment:
Why is the version required? It seems like things work without it -- at commit cd9608857, if I run `make dist` then webapp/vendor/composer/installed.php contains `'version' => 'dev-main'`; and if I first run `git tag 8.3.1` then it contains `'version' => '8.3.1.0'`.

Reply:
I'll check again, I got a local warning which I wanted to get rid of.

Reply:
Do you have any tips for spotting such warnings? (When I run the `make` commands there is a lot of output and I don't notice warnings; should I use an IDE?)

Reply:
I think `composer install` triggered it and otherwise `composer dump-autoload`.

I plan to work on this a bit more this week, so when I encounter it again I can pinpoint it a bit more.

What probably works is (from a clean checkout):
```
make dist | less
search for domjudge/domjudge
```

as it mentions that as package.

Reply:
I can't reproduce it with a clean checkout so my checkout might have been in a bad state.